client/checkout/stripe-link/index.js

```diff
@@ -4,6 +4,13 @@
 import { dispatchChangeEventFor } from '../utils/upe';
 
 export const switchToNewPaymentTokenElement = () => {
+	// Switch to card payment method before enabling new payment token element
+	document
+		.querySelector(
+			'input[name="payment_method"][value="woocommerce_payments"]'
+		)
+		?.click();
+
```

Comment:
It was 
```js
	document
		.querySelector( '.wcpay-upe-element[data-payment-method-type="card"]' )
		?.closest( '.wc_payment_method' )
		?.querySelector( 'input[type="radio"]' )
		?.click();
```
at first but after our discussion in https://github.com/Automattic/woocommerce-payments/pull/9782#discussion_r1862359667, I switched to 

```js
	document
		.querySelector(
			'input[name="payment_method"][value="woocommerce_payments"]'
		)
		?.click();
```